### PR TITLE
[feature/MORTAR-275] Update Promo Card component

### PR DESCRIPTION
**Pull request by bpearsoncse**
https://jira.nationalgeographic.com/browse/MORTAR-275

This ticket mostly revolves around accommodating the data modelings changes made in https://jira.nationalgeographic.com/browse/AEM-5758

I went ahead and created an Article component. Tickets to create subsequent components based on additional card types to come.

### CODE REVIEW

File: app/modules/promocard/scripts/MTPromoCard.jsx [hunk 1]
```diff
@@ -3,106 +3,90 @@
...
+
+  switch(props.type){
+    case 'article':
+      return <a {...attrs}>
+        <Article {...props} />
+      </a>;
+      break;
+    case 'video':
+      return <a {...attrs}>
+        <Article {...props} />
```
Comment (pmourelle) [inline app/modules/promocard/scripts/MTPromoCard.jsx]: video also uses `<Article>`?

**Comment by pmourelle** [comment]
LGTM.

File: app/modules/promocard/scripts/MTPromoCard.jsx (hunk 1 above)
Comment (bpearsoncse) [inline app/modules/promocard/scripts/MTPromoCard.jsx]: There is a separate ticket to create the video component. I was just testing the hide/show capability of the play button and forgot to delete. Will make change now.

**Comment by bpearsoncse** [comment]
Changes made. Back to you @pmourelle

**Pull request merged**